backend/testfiles/execution/language/epipe.dark

```diff
@@ -41,6 +41,7 @@ type X = { y: Y }
 
 type MyEnum = A of Int64 * Int64 * Int64
 (33L |> MyEnum.A 21L 42L) = MyEnum.A 33L 21L 42L
+(33L |> MyEnum.A(21L, 42L)) = MyEnum.A(33L, 21L, 42L)
```

Comment:
Ah, I don't like this. I think you should only be able to "pipe into an enum" when the enum has a _single_ field.
(maybe also with multiple fields, _if you pass in a tuple_)

Can we restrict things as such?

Reply:
To be clearer,

this are OK/legal:
`type MyEnum = | A of Bool` followed by `true |> MyEnum.A`

this is not:
`type MyEnum = | A of Bool * Bool` followed by `true |> MyEnum.A(false)`

this is maybe-ok, but fine to not implement: 
`type MyEnum = | A of Bool * Bool` followed by `(true, false) |> MyEnum.A`


---

Trying to do the "not ok" thing above is basically like saying:
`Tuple.addField (true,false) false` and expecting `(true, false, false)` which is a bit nonsensical as those are _totally different types_. I tried to do something similarly silly with tuples back in #4236. (enum fields are "tuple-like" in the same way, I think)